apps/.babelrc

```diff
@@ -2,8 +2,10 @@
   "presets": ["es2015", "react" ],
   "plugins": [
     "add-module-exports",
+    "syntax-async-functions",
```

Comment:
Neat!  How about enabling async/await support in its own PR?  It'd be a nice way to surface the new functionality to the team, and would cut back on this fairly large diff a little bit.

Reply:
Sounds good: https://github.com/code-dot-org/code-dot-org/pull/21513